sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -224,6 +243,8 @@ def _format_data(data: Union[str, IO]) -> Union[Tuple[None, str], Tuple[Optional
 
         :param data: The request field data.
         :type data: str or file-like object.
+        :rtype: tuple[None, str] or tuple[Optional[str], IO, str]
+        :return: A tuple of (data, content_type). If data is a string,
```

Comment:
Is this missing the part about different tuple structure if data is IO ?

Reply:
Fair